packages/conf-openssl/conf-openssl.1/files/osx-build.sh

```diff
@@ -1,6 +1,9 @@
 #!/bin/sh
 
-if [ -d "/usr/local/opt/openssl/" ]; then
+if [ -d "$HOME/.nix-profile/lib/pkgconfig/" ]; then
+  # Nix on macOS
+  PKG_CONFIG_PATH=$HOME/.nix-profile/lib/pkgconfig openssl
```

Comment:
Still not it, I think the correct line should be
```suggestion
  PKG_CONFIG_PATH=$HOME/.nix-profile/lib/pkgconfig pkg-config openssl
```